src/chrome/content/rules/RawGit.xml

```diff
@@ -0,0 +1,18 @@
+<!--
+	For RawGit.com
+
+		Works:
+
+			- $
+			- cdn
+
+		Doesn't Work:
+			- www (mismatch)
+-->
+<ruleset name="RawGit">
+  <target host="rawgit.com" />
+  <target host="*.rawgit.com" />
+
+  <rule from="^http://(?:(cdn\.)|www\.)?rawgit\.com/"
```

Comment:
Interesting, I wonder if it would be better to have two rules?
```xml
<rule from="^http://cdn\.rawgit\.com/"
          to="https://cdn.rawgit.com/" />

<rule from="^http://(?:www\.)?rawgit\.com/"
          to="https://$1rawgit.com/" />
```